docs/src/pages/components/data-grid/columns/RenderExpandCellGrid.js

```diff
@@ -120,7 +120,161 @@ renderCellExpand.propTypes = {
   /**
    * The column of the row that the current cell belongs to.
    */
-  colDef: PropTypes.any.isRequired,
+  colDef: PropTypes.shape({
```

Comment:
this looks way too verbose for a demo

Reply:
This line has been generated, but I guess it's an error in the TS => PropTypes generator.